### ISSUE TITLE
Synonyms used in mapping should be visible to the user in some way

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/opentree#499]
For example in commit,
https://github.com/OpenTreeOfLife/phylesystem-1/commit/dfd9503d8706c33d814d253f0f455ef9c798b466

I modified 'Eriocran' to 'Eriocrania subpurpurella' then when I clicked the map, it resolved to "Dyseriocrania subpurpurella" in OTT.

That mapping is the correct. "Eriocrania subpurpurella" is a junior synonym (suppl. mat. of the paper confirms this).

But it would be nice if the user could see that the match was actually a perfect match, and the discrepancy is due to synonymy. The TNRS did not just make a "close enough" match of the text strings. 

In this case it is not obvious because the valid name is only 3 letters off from the junior synonym. But (due to taxonomists tendencies to reuse Greek/Latin roots to generate very similar genus names, and the preservation of the specific epithet in most reclassifications) this sort of case won't be very rare.

*edited to clarify that the "suppl. mat." refers to info from the supplementary materials of the publication.

**Comment by mtholder** [comment]
@jimallman looks great to me. 
My (slight) preference would be that we would find a way to add the matched name (the junior synonym) to the nexson as an optional field.  That would make it easier for people who are working with the corpus of NexSONs to understand what happened. But that is a minor concern. Perhaps we can discuss it on the next call, and make an issue for that feature if we decide that it is wise.

**Comment by jimallman** [comment]
Agreed, let's discuss and (if desired) make a separate issue for adding this to nexson. Keeping this open as a reminder to have that discussion. :grin:  Maybe discuss on the **opentree** (or **opentree-software**) group?
